samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -67,6 +74,11 @@ static struct bt_le_scan_cb scan_callbacks = {
 	.recv = scan_recv,
 };
 
+static uint32_t sync_create_timeout_get(uint16_t interval, uint8_t timeout_interval_num)
+{
+	return BT_GAP_ADV_INTERVAL_TO_MS(interval) * timeout_interval_num;
+}
```

Comment:
```suggestion
static uint32_t sync_create_timeout_get(uint16_t interval)
{
	return BT_GAP_ADV_INTERVAL_TO_MS(interval) * SYNC_CREATE_TIMEOUT_INTERVAL_NUM;
}
```

Reply:
If you are only supplying a macro'ed value, you might as well just use that directly in this function.

Reply:
Changed, even though it would have been optimized during compilation.